### DISCUSSION TITLE
select all checkbox need a label

### DISCUSSION THREAD

**Discussion by niks1129** [primefaces/community#997] [PrimeReact] [answered]
select all checkbox need a label in multiselect component.

**Comment by melloware** [comment, marked as answer]
It has one: https://stackblitz.com/edit/cweb3f?file=src%2FApp.jsx

See https://primereact.org/multiselect/#api.MultiSelect.props.selectAllLabel

**Reply by niks1129** [reply]
with filter it should work but not working on your demo and also I'm not able to access selectAllLabel attached screenshot below.

**Reply by melloware** [reply]
Make sure you are using latest PrimeReact.  It won't work with filter.

**Comment by niks1129** [comment]
![image](https://github.com/primefaces/community/assets/54804767/41935b26-ba43-44e0-ac2a-2bc1381a3e5a)
